scripts/inference/cropland_mapping.py

```diff
@@ -1,21 +1,19 @@
 """Cropland mapping inference script, demonstrating the use of the GFMAP, Presto and WorldCereal classifiers in a first inference pipeline."""
 
 import argparse
+from pathlib import Path
 
+import openeo
 from openeo_gfmap import BoundingBoxExtent, TemporalContext
-from openeo_gfmap.backend import Backend, BackendContext, cdse_connection
-from openeo_gfmap.features.feature_extractor import PatchFeatureExtractor
+from openeo_gfmap.backend import Backend, BackendContext
+from openeo_gfmap.features.feature_extractor import apply_feature_extractor
+from openeo_gfmap.inference.model_inference import apply_model_inference
 
+from worldcereal.openeo.feature_extractor import PrestoFeatureExtractor
+from worldcereal.openeo.inference import CroplandClassifier
 from worldcereal.openeo.preprocessing import worldcereal_preprocessed_inputs_gfmap
 
-
-class PrestoFeatureExtractor(PatchFeatureExtractor):
-    def __init__(self):
-        pass
-
-    def extract(self, image):
-        pass
-
+ONNX_DEPS_URL = "https://artifactory.vgt.vito.be/artifactory/auxdata-public/openeo/onnx_dependencies_1.16.3.zip"
```

Comment:
Why is this still needed? `onnxruntime` is included in the `worldcereal_deps.zip`. And if it's still needed, shouldn't we also put it on s3?

Reply:
Because it's much lighter than all the presto dependency so it's a waste of resources to install everything just for `onnxruntime`. Also it's a requirement in GFMAP for the ModelInference subclasses, until the pip dependency feature is more stable.

Indeed, could you updload it to the S3 @HansVRP ?

Reply:
Does this mean that for each single UDF in the chain those dependencies get downloaded/unpacked again? That's also lots of overhead I guess?

Reply:
In this case it's just onnxruntime that get's reinstalled, compared to the full UDF I don't think there is a large difference in execution time, even less with large spatial extents probably